pyproject.toml

```diff
@@ -10,7 +10,7 @@ readme = "README.md"
 requires-python = ">=3.10"
 dependencies = [
     "python-dotenv >=1.0.0",
-    "cognite-sdk >=7.80.2, <8.0.0",
+    "cognite-sdk >=7.81.3, <8.0.0",
```

Comment:
This is the version of the `cognite-sdk` that allows setting the `nonce` directly.